DEVELOPER_GUIDE.md

```diff
@@ -49,7 +49,7 @@ Then you will find the built artifact located at `build/distributions` directory
 ## Install
 Once you have built the plugin from source code, run
 ```bash
-opensearch-plugin install file://${PLUGIN_ZIP_FILE_PATH}
+opensearch-plugin install file://build/distributions/opensearch-job-scheduler-1.1.0.0-SNAPSHOT.zip
```

Comment:
I believe this is incorrect, it should be reverted. If this is failing link checker we might need to skip this file checked(?)

Reply:
it looks like all the plugins removed install section from the DEVELOPER_GUIDE. 

I see one example in security plugin which tries to specify the path, you can try this to see if setting file path to something like this prevent link checker failure - https://github.com/opensearch-project/security-dashboards-plugin/blob/main/DEVELOPER_GUIDE.md#install